src/doc/trpl/enums.md

```diff
@@ -2,67 +2,76 @@
 
 Rust has a ‘sum type’, an `enum`. Enums are an incredibly useful feature of
 Rust, and are used throughout the standard library. An `enum` is a type which
-relates a set of alternates to a specific name. For example, below we define
-`Character` to be either a `Digit` or something else.
+relates a set of alternates to a specific name. Each variant of an
+`enum` is defined like a struct, and can hold any data (or no data, like
+a unit-like struct).
```

Comment:
I'm not sure how I feel about this characterization. I can see where you're coming from, but these things are very separate in my brain.